core/src/test/scala/org/apache/spark/deploy/history/FsHistoryProviderSuite.scala

```diff
@@ -43,7 +43,7 @@ class FsHistoryProviderSuite extends FunSuite with BeforeAndAfter with Matchers
     testDir = Utils.createTempDir()
     provider = new FsHistoryProvider(new SparkConf()
       .set("spark.history.fs.logDirectory", testDir.getAbsolutePath())
-      .set("spark.history.fs.updateInterval", "0"))
+      .set("spark.testing", "true"))
```

Comment:
Hmm... as Sean mentioned, this should already be defined [here](https://github.com/apache/spark/blob/master/pom.xml#L1127). Can you double check that it's really not set, and if not, what's causing it?